src/formatter/formatPropValue.spec.js

```diff
@@ -0,0 +1,127 @@
+/* @flow */
+
+import React from 'react';
+import formatPropValue from './formatPropValue';
+import parseReactElement from './../parser/parseReactElement';
+import formatTreeNode from './formatTreeNode';
+import formatComplexDataStructure from './formatComplexDataStructure';
+
+jest.mock('./../parser/parseReactElement');
+jest.mock('./formatTreeNode', () =>
+  jest.fn().mockReturnValue('<MockedFormatTreeNodeResult />')
+);
+jest.mock('./formatComplexDataStructure', () =>
+  jest.fn().mockReturnValue('*Mocked formatComplexDataStructure result*')
+);
+
+describe('formatPropValue', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should format an integer prop value', () => {
+    expect(formatPropValue(42, false, 0, {})).toBe('{42}');
+  });
+
+  it('should escape double quote on prop value of string type', () => {
+    expect(formatPropValue('Hello "Jonh"!', false, 0, {})).toBe(
+      '"Hello &quot;Jonh&quot;!"'
+    );
+  });
+
+  it('should format a symbol prop value', () => {
+    expect(formatPropValue(Symbol('Foo'), false, 0, {})).toBe('{Symbol(Foo)}');
+  });
+
+  it('should replace a function prop value by a an empty generic function by default', () => {
+    const doThings = a => a * 2;
+
+    expect(formatPropValue(doThings, false, 0, {})).toBe(
+      '{function noRefCheck() {}}'
+    );
+  });
+
+  it('should show the function prop value implementation if "showFunctions" option is true', () => {
+    const doThings = a => a * 2;
+
+    expect(formatPropValue(doThings, false, 0, { showFunctions: true })).toBe(
+      '{function doThings(a) {return a * 2;}}'
+    );
+  });
+
+  it('should format the function prop value with the "functionValue" option', () => {
+    const doThings = a => a * 2;
+
+    const functionValue = fn => {
+      expect(fn).toBe(doThings);
+
+      return 'function Myfunction() {}';
+    };
+
+    expect(
+      formatPropValue(doThings, false, 0, {
+        functionValue,
+        showFunctions: true,
+      })
+    ).toBe('{function Myfunction() {}}');
+
+    expect(
+      formatPropValue(doThings, false, 0, {
+        functionValue,
+        showFunctions: false,
+      })
+    ).toBe('{function Myfunction() {}}');
+  });
+
+  it('should parse and format a react element prop value', () => {
+    expect(formatPropValue(<div />, false, 0, {})).toBe(
+      '{<MockedFormatTreeNodeResult />}'
+    );
+
+    expect(parseReactElement).toHaveBeenCalledTimes(1);
+    expect(formatTreeNode).toHaveBeenCalledTimes(1);
+  });
+
+  it('should format a date prop value', () => {
+    expect(
+      formatPropValue(new Date('2017-01-01T11:00:00.000Z'), false, 0, {})
+    ).toBe('{new Date("2017-01-01T11:00:00.000Z")}');
```

Comment:
Yes I guess this is one. 

The escaping of the brace in string could be considered as breaking too (see f14ba62)